config/settings/development.yml

```diff
@@ -1,6 +1,12 @@
 # The url for the google doc feedback link (non-live version)
 feedback_link_url: "https://docs.google.com/forms/d/e/1FAIpQLSdYg4IA2tMiSfqIjM9CslVgFzsSazznD6VoT41g19sJYCwsLQ/viewform"
 
+# The url for the google doc feedback link for recommended for award (non-live version)
```

Comment:
The non-live version should be the default for all environments except for prod. Then for prod we explicitly set the live version. It means two settings file are modified instead of all.

Reply:
Have removed the references from all yml files and just left non-live links in `settings.yml` and live links in `production.yml`